src/core/Directus/Filesystem/Files.php

```diff
@@ -1,7 +1,9 @@
 <?php
 
 namespace Directus\Filesystem;
+require_once '/var/www/html/vendor/autoload.php';
```

Comment:
This path might not work for all the users.
Can we use the char0n/ffmpeg-php via composer as mentation https://github.com/char0n/ffmpeg-php#composer-installation?